ltsp/common/ltsp/55-images.sh

```diff
@@ -23,6 +23,8 @@ img_path_to_name() {
     img_path=$1
     img_name=${img_path##*/}
     if [ -f "$img_path" ]; then
+        # QUESTION: why do images need to be named *.img?
+        # Wouldn't it work just as well if images could have arbitrary names?
```

Comment:
Because someone might want a /srv/ltsp/images/README.md that shouldn't be considered an image.

Reply:
Forgot that in there ;-)
I ended up going the *.img way anyways.